dynamiq/callbacks/tracing.py

```diff
@@ -477,6 +477,9 @@ def on_node_execute_run(self, serialized: dict[str, Any], **kwargs: Any):
         if prompt_messages := kwargs.get("prompt_messages"):
             run.metadata["node"]["prompt"]["messages"] = prompt_messages
 
+        if session_id := kwargs.get("session_id"):
+            run.metadata["session_id"] = session_id
```

Comment:
Is this a clear name?
We already have `session_id` on top of the Run object. So maybe add some prefix